plugins/jest.js

```diff
@@ -67,9 +67,9 @@ module.exports = {
     'jest/prefer-spy-on': 0,
     // suggest using `toStrictEqual()`
     'jest/prefer-strict-equal': 1,
-    // suggest using `toBeNull()`
+    // Deprecated: suggest using `toBeNull()`
     'jest/prefer-to-be-null': 2,
-    // suggest using `toBeUndefined()`
+    // Deprecated: suggest using `toBeUndefined()`
     'jest/prefer-to-be-undefined': 2,
```

Comment:
Let's just nuke these deprecations in this breaking change since you've got the `prefer-to-be` added.

Reply:
Sounds good, will update.